src/longopt/dimfilter_inc.h

```diff
@@ -24,39 +24,20 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 	/* separator, short_option, long_option,
 	          short_directives,    long_directives,
 	          short_modifiers,     long_modifiers */
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'I', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'N', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'Q', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'S', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'T', "",
-	          "",                  "",
-	          "",                  "" },
+	{ 0, 'D', "distance",     "", "", "", "" },
+	{ 0, 'F', "filter",
+	          "b,c,g,m,p",    "boxcar,cosarch,gaussian,median,maxprob",
+	          "l,u",          "lower,upper" },
+	{ 0, 'G', "outgrid",
+	          "",             "",
+	          "d,n,o,s",      "divide,nan,offset,scale" },
+	GMT_INCREMENT_KW,         /* quasi-common -I option from gmt_constants.h */
+	{ 0, 'L', "stdoutscript", "", "", "", "" },
```

Comment:
Most people don't know what is ``stdout``. Better if we call it ``bashscript``, which is what it does.

Reply:
Agree missed that one

Reply:
Chose `--script `since may not be bash only (even though it is now).